app/views/admins/dashboard.mobile.haml

```diff
@@ -0,0 +1,4 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+= render partial: "admins/dashboard"
```

Comment:
This has the `.hidden-xs` class so it is not displayed on mobile. But you're right this hack is not nice, because large devices on mobile version still have the message, and mobile devices with desktop version don't have it anymore. I'll do it with CSS to hide it if the user is using the mobile version, not depending on his screen size.